File: Java/program70.c
///////////////////////////////////////////////////////
//
//  File name :     program70.c
//  Description :   Accept Input Number and Display Even Digits from that Number
//  Author :        Vaibhav
//  Date :          20/05/2025
//
///////////////////////////////////////////////////////

#include<stdio.h>

void DisplayEven(int iValue)
{
    int iDigit = 0;

    if(iValue < 0)
    {
        iValue = -iValue;
    }
    while(iValue != 0)
    {
        iDigit = iValue % 10;
        
        if( (iDigit % 2) == 0)
        {
            printf("%d\n",iDigit);
        }
        iValue = iValue / 10;
    }
}

int main()
{
    int iNo = 0;
    int iRet = 0;

    printf("---- This Application is used to Display Even Digits from that Number ----\n");

    printf("Enter Number: ");
    scanf("%d",&iNo);

    printf("Even Digits in Number %d are : \n",iNo);
    DisplayEven(iNo);

    return 0;
}

File: program70.java
///////////////////////////////////////////////////////
//
//  File name :     program70.java
//  Description :   Accept Input Number and Display Even Digits from that Number
//  Author :        Vaibhav
//  Date :          
//
///////////////////////////////////////////////////////

import java.util.*;

class Digits
{
    public void DisplayEven(int iNo)
    {
        int iDigit = 0;
        
        if(iNo < 0)
        {
            iNo = -iNo;
        }
        while(iNo != 0)
        {
            iDigit = iNo % 10;              //for digit separation
            if((iDigit % 2) == 0)
            {
                System.out.println(iDigit);
            }
            iNo = iNo / 10;                 //
        }
    }
}

class program70
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("---- This Application is used to Display Even Digits from that Number ----");        
        
        System.out.println("Enter a number: ");
        int iValue = sobj.nextInt();

        System.out.println("Even Digits in Number "+iValue+" are : ");
        Digits dobj = new Digits();
        
        dobj.DisplayEven(iValue);

    }
}